common/src/primitives/height.rs

```diff
@@ -18,7 +18,7 @@ use std::ops::{Add, Sub};
 
 use serialization::{Decode, Encode};
 
-type HeightIntType = u64;
+pub type BlockHeightIntType = u64;
```

Comment:
I would prefer to keep this private. I think the better way to do this is to keep this private, and in the database, use u64, and static_assert on it that it's u64 if necessary, and not another type.

Reply:
> I would prefer to keep this private.

Why?

Btw, the similar `BlockTimestampInternalType` is public.